### PR TITLE
Mission control url

### PR DESCRIPTION
**Pull request by edewit**
added mission control url and connected the websockets to show the status list

to test fire up mission-control https://github.com/openshiftio/launchpad-missioncontrol/pull/216 (on port 8080 or configure `LAUNCHPAD_MISSION_CONTROL_URL`) and backend (on port 8180 or configure `LAUNCHPAD_BACKEND_URL`) locally. Launch a booster and see the status list

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -7,14 +7,16 @@ var helpers = require('./helpers');
 const ENV = process.env.ENV || process.env.NODE_ENV || 'development';
 // if env is 'inmemory', the inmemory debug resource is used
 const API_URL = process.env.API_URL || (ENV==='inmemory'?'app/':'http://localhost:8080/api/');
-const LAUNCHPAD_BACKEND_URL = process.env.LAUNCHPAD_BACKEND_URL || 'http://localhost:8080/';
+const LAUNCHPAD_BACKEND_URL = process.env.LAUNCHPAD_BACKEND_URL || 'http://localhost:8180/';
```
Comment (quintesse) [inline config/webpack.dev.js]: localhost:8180 ? Isn't it 8081?

Comment (edewit) [inline config/webpack.dev.js]: does it matter? I do `-Dswarm.port.offset=100`

**Comment by quintesse** [comment]
And as a general question because of something @gastaldi and I discussed yesterday: does the LAUNCHPAD_MISSIONCONTROL_URL have a path part associated to it? Eg. do you connect directly to `ws://localhost:8080` or something like `ws://localhost:8080/status`?
I ask because of the reverse proxying.

File: config/webpack.dev.js (hunk 1 above)
Comment (quintesse) [inline config/webpack.dev.js]: Ah no, but in the docs we suggest to use offset=1, right?

File: src/app/wizard/deploy/deploy.component.ts [hunk 2]
```diff
@@ -11,52 +12,64 @@ import {KeycloakService} from "../../shared/keycloak.service";
...
-    this.kc.login();
-  }
-
-  isAuthenticated():boolean {
-    return this.kc.isAuthenticated();
+    if (this.kc.isAuthenticated()) {
+      this.progress = true;
+      this.forgeService.upload(this.command, this.submittedGuis)
+        .then(status => {
+          this.webSocket = new WebSocket(this.apiUrl + status.uuid_link);
```
Comment (edewit) [inline src/app/wizard/deploy/deploy.component.ts]: @quintesse yes connect to `ws://localhost:8080/status/<guid>` the `StatusResult` contains that url part. So if we have reverse proxy the `LAUNCHPAD_MISSIONCONTROL_URL` can be empty

**Comment by quintesse** [comment]
Squash-merged. Closing.

**Pull request closed without merging**